README.md

```diff
@@ -7,6 +7,16 @@ Common code shared among
 [MySQL](https://github.com/juttle/juttle-mysql-adapter/)
 and [SQLite](https://github.com/juttle/juttle-sqlite-adapter/) adapters.
 
+#### List of optimized operations
+
+* any filter expression `read sql` (note: `read sql | filter ...` is not optimized)
```

Comment:
Ok, I put in a ticket for this: https://github.com/juttle/juttle-sql-adapter-common/issues/37. Sql Adapter does not support RegExp, just `~=` globs.